src/test/java/edu/ksu/canvas/attendance/submitter/AssignmentSubmitterUTest.java

```diff
@@ -123,34 +123,34 @@ public void setup() {
         attendanceSummaryModelList.add(attendanceSummaryModel2);
 
         assignmentConfigurationFromSetup = new AttendanceAssignment();
-        assignmentConfigurationFromSetup.setAbsentPoints(0.0);
-        assignmentConfigurationFromSetup.setAssignmentPoints(ASSIGNMENT_POINTS);
-        assignmentConfigurationFromSetup.setExcusedPoints(0.0);
-        assignmentConfigurationFromSetup.setPresentPoints(100.0);
-        assignmentConfigurationFromSetup.setTardyPoints(0.0);
+        assignmentConfigurationFromSetup.setAbsentPoints("0.0");
+        assignmentConfigurationFromSetup.setAssignmentPoints(String.valueOf(ASSIGNMENT_POINTS));
+        assignmentConfigurationFromSetup.setExcusedPoints("0.0");
+        assignmentConfigurationFromSetup.setPresentPoints("100.0");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "100.0" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)